src/zerocopy.rs

```diff
@@ -43,18 +57,28 @@ impl<T> Sender<T>
 where
     T: OwningMessage,
 {
+    /// Sends a non-owning message to the receiver in this channel.
+    ///
+    /// If the receiver is currently waiting for a message, the non-owning data
+    /// will be sent as-is. Otherwise, the data is converted to its owned
+    /// version and queued for later.
     pub fn send<'a>(&self, data: T::NonOwning<'a>) -> Result<SendFut<'a>, SendError<T>> {
         let owned = data.to_owned();
         self.inner_tx.send(owned)?;
         let fut = SendFut { _lock: PhantomData };
         Ok(fut)
     }
 
+    /// The number of receivers in this channel.
```

Comment:
This is a function. Its doc comment should be a verb. Or the name should just be `get_reciever_count` and there shouldnt be a doc comment at all.

Reply:
yup:
```
Get the number of receivers in this channel.
```